.github/workflows/publish-pr-test-results.yaml

```diff
@@ -35,4 +35,5 @@ jobs:
           commit: ${{ github.event.workflow_run.head_sha }}
           event_file: artifacts/Event File/event.json
           event_name: ${{ github.event.workflow_run.event }}
-          files: "**/target/surefire-reports/**/*.xml"
\ No newline at end of file
+          files: |
```

Comment:
Why breaking over the line suddenly?

Reply:
I was considering multiple paths.  (At least now, it's consistent with the other, similar clauses in the yaml files.)